src/mongo/Config.class.php

```diff
@@ -2176,4 +2160,30 @@ public static function getLogger()
         }
         return self::$logger;
     }
+
+    /**
+     * Sets the Tripod config
+     *
+     * @param array $config
+     * @return Config
+     */
+    public static function deserialize(array $config)
+    {
+        if (isset($config['class']) && isset($config['config'])) {
```

Comment:
The idea is that the current config (the default here) would be deprecated to avoid ambiguity as well as any collisions with the `class` property.  So the default config would look like:
```
{
   "class" => "\Tripod\Mongo\Config",
   "config" => { ... what we're currently passing as $config ... }
}
```